Cargo.toml

```diff
@@ -89,10 +89,138 @@ members = [
 ]
 resolver = "2"
 
-[patch]
-[patch.crates-io]
-# TODO: remove when 0.3.6 get's released.
-warp = { git = "https://github.com/seanmonstar/warp.git", rev="149913fe" }
+[workspace.package]
+edition = "2021"
+
+[workspace.dependencies]
+arbitrary = { version = "1", features = ["derive"] }
+bincode = "1"
+bitvec = "1"
+byteorder = "1"
+bytes = "1"
+clap = "2"
+compare_fields_derive = { path = "common/compare_fields_derive" }
+criterion = "0.3"
+delay_map = "0.3"
+derivative = "2"
+dirs = "3"
+discv5 = { version = "0.3", features = ["libp2p"] }
+env_logger = "0.9"
+error-chain = "0.12"
+ethereum-types = "0.14"
+ethereum_hashing = "1.0.0-beta.2"
+ethereum_serde_utils = "0.5"
+ethereum_ssz = "0.5"
+ethereum_ssz_derive = "0.5"
+ethers-core = "1"
+ethers-providers = { version = "1", default-features = false }
+exit-future = "0.2"
+fnv = "1"
+fs2 = "0.4"
+futures = "0.3"
+hex = "0.4"
+hyper = "0.14"
+itertools = "0.10"
+lazy_static = "1"
+libsecp256k1 = "0.7"
+log = "0.4"
+lru = "0.7"
+maplit = "1"
+num_cpus = "1"
+parking_lot = "0.12"
+paste = "1"
+quickcheck = "1"
+quickcheck_macros = "1"
+quote = "1"
+r2d2 = "0.8"
+rand = "0.8"
+rayon = "1.7"
+regex = "1"
+reqwest = { version = "0.11", features = ["blocking", "json", "stream", "native-tls-vendored"] }
```

Comment:
We can (and should) turn off the `blocking` feature now that https://github.com/sigp/lighthouse/pull/4753 is merged

Reply:
yeah, thanks Michael, I was waiting for https://github.com/ralexstokes/ethereum-consensus/pull/261 to then switch `native-tls-vendored` for just `rustls-tls` but we can do that on a subsequent PR. Updated!

Reply:
https://github.com/sigp/lighthouse/actions/runs/6268152351/job/17022546605?pr=4650#step:8:54 seems to fail without `blocking`, reverted. I can address it in a subsequent PR so as to not block this one

Reply:
oh yeah no worries!